src/lib/crypto/kimchi_bindings/stubs/src/lagrange_basis.rs

```diff
@@ -2,7 +2,9 @@ use ark_ec::AffineRepr;
 use ark_poly::{EvaluationDomain, Radix2EvaluationDomain as D};
 use cache::LagrangeCache;
 use mina_curves::pasta::{Pallas, Vesta};
-use poly_commitment::{commitment::CommitmentCurve, srs::SRS};
```

Comment:
The `.rustfmt.toml` in `proof-systems` would prefer the style that you removed. I guess it's not important rn but let's unify it later. Maybe if you just drop a `.rustfmt.toml` from `proof-systems` into the `mina` root it'll just work.

Reply:
This will be fixed when moving into proof-systems.